build.sh

```diff
@@ -22,11 +22,14 @@ case "${PODMAN_PR_NUM}" in
   *) echo 'PODMAN_PR_NUM must be empty or set to a valid PR number' 1>&2; exit 1;;
 esac
 
-# See podman-rpm-info-vars.sh for all build-arg values. If PODMAN_PR_NUM is
-# empty, the rpm version, release and fedora release values are of no concern
-# to the build process.
+# See podman-rpm-info-vars.sh PODMAN_PR_NUM value. If PODMAN_PR_NUM is
+# empty, other variables are of effectively no concern to the build process.
+dnf -y install python3-fedora-distro-aliases
```

Comment:
This should be added in the CI image directly.

Reply:
done